.github/workflows/generateTranslations.yml

```diff
@@ -47,6 +47,10 @@ jobs:
         with:
           ref: ${{ steps.pr-data.outputs.HEAD_SHA }}
 
+      - name: Fetch PR head ref
+        run: |
+          git fetch --no-tags --depth=0 origin +refs/pull/${{ steps.pr-data.outputs.PR_NUMBER }}/head:refs/remotes/origin/pr/${{ steps.pr-data.outputs.PR_NUMBER }}
```

Comment:
It's unclear to me why this would not be the same as the previous step's checkout w/ ref set to the PR HEAD.